recipe/meta.yaml

```diff
@@ -10,36 +10,29 @@ source:
   sha256: 069051f3cf30bda3f3c301682855ec0b40e00b3ebce0f20f7f10f765465eb308
 
 build:
-  number: 1
-  skip: True  # [py<38]
+  number: 2
+  noarch: python
   script:
     - {{ PYTHON }} setup.py bdist_wheel --dist-dir=dist
     - {{ PYTHON }} -m pip install --find-links=dist --no-deps --ignore-installed --no-cache-dir -vvv {{ name }}
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.8
     - pip
     - setuptools
     - setuptools_scm
     - setuptools_scm_git_archive
     - wheel
   run:
-    - python
+    - python  >=3.8
     - cftime
     - cgen
     - dask
-    - enum34  # [py2k]
-    - gcc_linux-64  # [linux]
-    - clang_osx-64  # [osx]
-    - m2w64-toolchain  # [win]
+    - enum34
```

Comment:
Can remove [`enum34`](https://pypi.org/project/enum34/), this is for Python<=3.3 only.
```suggestion
```